CMakeLists.txt

```diff
@@ -131,6 +131,17 @@ foreach(optional_include_dir
   endif()
 endforeach()
 
+# Don't inherit compiler flags from the environment
+foreach(var AFLAGS CFLAGS CXXFLAGS CPPFLAGS LDFLAGS)
+  if(DEFINED ENV{${var}})
+    message(WARNING "The environment variable '${var}' was set to $ENV{${var}}, "
+                    "but Zephyr ignores flags from the environment. Use 'cmake "
+		    "-DEXTRA_${var}=$ENV{${var}}' instead."
```

Comment:
Alignment here is off

Reply:
note to self.
Check formatting when copying other peoples code to new location 😆 

Fixed.